website/docs/r/db_proxy.html.markdown

```diff
@@ -0,0 +1,86 @@
+---
+subcategory: "RDS"
+layout: "aws"
+page_title: "AWS: aws_db_proxy"
+description: |-
+  Provides an RDS DB proxy resource.
+---
+
+# Resource: aws_db_proxy
+
+Provides an RDS DB proxy resource. For additional information, see the [RDS User Guide](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/rds-proxy.html).
+
+## Example Usage
+
+```hcl
+resource "aws_db_proxy" "example" {
+  name                   = "example"
+  debug_logging          = false
+  engine_family          = "MYSQL"
+  idle_client_timeout    = 1800
+  require_tls            = true
+  role_arn               = "arn:aws:iam:us-east-1:123456789012:role/example"
+  vpc_security_group_ids = ["sg-12345678901234567"]
+  vpc_subnet_ids         = ["subnet-12345678901234567"]
+
+  auth {
+    auth_scheme = "SECRETS"
+    description = "example"
+    iam_auth    = "DISABLED"
+    secret_arn  = "arn:aws:secretsmanager:us-east-1:123456789012:secret:example"
```

Comment:
Let's show example resource references here instead of hardcoded values, which would be atypical (and IAM Role ARNs do not have a region 😉 ). 👍 

```suggestion
  role_arn               = aws_iam_role.example.arn
  vpc_security_group_ids = [aws_security_group.example.id]
  vpc_subnet_ids         = [aws_subnet.example.id]

  auth {
    auth_scheme = "SECRETS"
    description = "example"
    iam_auth    = "DISABLED"
    secret_arn  = aws_secretsmanager_secret.example.arn
```